core-plugins/docs/Distinct-batchaggregator.md

```diff
@@ -15,6 +15,9 @@ Properties
 ----------
 **fields:** Optional comma-separated list of fields to perform the distinct on. If not given, all fields are used.
 
+**numPartitions:** Number of partitions to user when grouping fields. If not specified, the execution
```

Comment:
nice. this happens a lot to me for some reason. fingers autocorrect use to user.